src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -142,12 +142,18 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
           }
           break;
         case FORMAT_NAME_NODES :
-          if (!liveState.isNameNode1Initialized()) {
+          if (!liveState.isNameNode1Initialized() && !liveState.isNameNode2Initialized()) {
             sendMessageTo(
                 driver,
                 liveState.getFirstNameNodeTaskId(),
                 liveState.getFirstNameNodeSlaveId(),
                 HDFSConstants.NAME_NODE_INIT_MESSAGE);
+          } else if (!liveState.isNameNode1Initialized()) {
```

Comment:
Aren't line 145 and 151 the same if statement? i think 151 should be:
          } else if (liveState.isNameNode1Initialized()) {